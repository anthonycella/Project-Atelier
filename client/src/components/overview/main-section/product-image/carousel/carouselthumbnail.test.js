import renderer from 'react-test-renderer';
import CarouselThumbnail from './carouselthumbnail.jsx';

it('Carousel Thumbnail renders to the page', () => {
  const component = renderer.create(
    <CarouselThumbnail />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

it('Carousel Thumbnail works for fullscreen too', () => {
  const component = renderer.create(
    <CarouselThumbnail fullScreen={true} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
