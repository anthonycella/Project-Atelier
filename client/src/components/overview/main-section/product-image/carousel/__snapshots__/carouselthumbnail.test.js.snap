// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel Thumbnail renders to the page 1`] = `
<div
  className="sc-gsnTZi dEGviv"
>
  <div
    className="sc-bczRLJ cIQPcm"
    height={60}
    width={65}
  />
</div>
`;

exports[`Carousel Thumbnail works for fullscreen too 1`] = `
<div
  className="sc-gsnTZi dEGviv"
>
  <div
    className="sc-bczRLJ cIQPcm"
    height={60}
    width={65}
  />
</div>
`;
